Testing/SystemTests/tests/framework/SANSILLReductionTest.py

```diff
@@ -169,8 +169,7 @@ def runTest(self):
         SANSILLReduction(Run='093410', ProcessAs='Sample', BeamInputWorkspace='beam', TransmissionInputWorkspace='str',
                          ContainerInputWorkspace='can', MaskedInputWorkspace='mask', OutputWorkspace='sample', FluxInputWorkspace='flux')
         # I(Q)
-        SANSILLIntegration(InputWorkspace='sample', OutputBinning='0.005,-0.1,1',
-                           OutputWorkspace='iq', BinMaskingCriteria='x<1 || x>10')
+        SANSILLIntegration(InputWorkspace='sample', OutputBinning='0.005,-0.1,1', OutputWorkspace='iq')
```

Comment:
The default value of the lambda range provides the same outcome as the former `BinMaskingCriteria` but perhaps it would be nice to have the property set explicitly in the test?

Reply:
I favour specifying only what is not default, as it is clearer. It's actually better for this test to check that the default is fine.